lib/fog/proxmox/helpers/nic_helper.rb

```diff
@@ -106,7 +106,10 @@ def self.flatten(nic_hash)
           end
           nic_value += ',' + Fog::Proxmox::Hash.stringify(options) unless options.empty?
         else
-          nic_value = Fog::Proxmox::Hash.stringify(nic_hash.except(%i[id model macaddr])) unless nic_hash.empty?
```

Comment:
is there a test for the "flatten" method?

Reply:
Added the test in foreman_fog_proxmox: https://github.com/theforeman/foreman_fog_proxmox/pull/338